app/app.rb

```diff
@@ -0,0 +1,66 @@
+require "sinatra/base"
+require "sinatra/json"
+require "./lib/menu"
+require "./lib/order_list"
+require "./lib/order"
+require "./lib/tax"
+require "./lib/discount"
+require "./lib/receipt"
+require "./lib/payment"
+require "./app/modules/my_helpers"
+
+class TillTechTest < Sinatra::Base
+  set :root, File.dirname(__FILE__)
+  alias_method :s, :settings
+  helpers Sinatra::JSON
+  include MyHelpers
+  set :static, true
+  set :order_list, OrderList.new
+  set :menu,       Menu.new([{ name: "Cafe Latte", price: 2.5 },
```

Comment:
Set is used for setting configuration values, don't use it to sneak in global objects into sinatra! Just define them where needed in your controller, if you find yourself repeating code, that shows you'll need a factory method or a service object